lib/Rest/GeneralResponse.h

```diff
@@ -136,6 +136,8 @@ class GeneralResponse {
     _headers.emplace(key, value);
   }
 
+  virtual bool isResponseEmpty() const {return false;}
```

Comment:
Shouldn't this method be pure virtual?
IMHO having a default implementation makes no sense.

Reply:
it did - it uncovered that the mock didn't override it, since the assertion kicked in.